Packs/Infinipoint/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### Infinipoint 
+- Upgrade of test playbook
+- Fixed formatting 
+- Adding explanations
```

Comment:
Use our default format for this change:
- Documentation and metadata improvements.


Please take a look here https://xsoar.pan.dev/docs/integrations/release-notes and change the release notes accordingly.